parsers/mysql/mysql.go

```diff
@@ -307,19 +312,19 @@ func (p *Parser) handleEvents(rawEvents <-chan rawEvent, send chan<- event.Event
 func (p *Parser) handleEvent(rawE rawEvent) SlowQuery {
 	var err error
 	sq := SlowQuery{}
-	logrus.WithFields(logrus.Fields{
-		"rawE": rawE,
-	}).Debug("About to parse event")
+	query := ""
 	for _, line := range rawE.lines {
 		// parse each line and populate the SlowQuery object
 		switch {
 		case reTime.MatchString(line):
+			query = ""
```

Comment:
why do you zero out query on each match? Wouldn't it be enough that it starts out empty and gets filled only by the reQuery switch case?
aaaaand query is only ever used within the reQuery case. Skip all this and just declare it local to that block?